src/dataLabel/dataLabelUtils.ts

```diff
@@ -221,10 +221,12 @@ export function drawDefaultLabelsForDataPointChart(data: any[], context: Selecti
             .duration(animationDuration)
             .style(layout.style as any)
             .style("opacity", (hasSelection ? (d: SelectableDataPoint) => getFillOpacity(d.selected, false, hasSelection, false) : 1) as any)
-            .attr({
-                x: (d: LabelEnabledDataPoint) => d.labelX,
-                y: (d: LabelEnabledDataPoint) => d.labelY
-            });
+            .attr(
+                "x", (d: LabelEnabledDataPoint) => d.labelX
+            )
+            .attr(
+                "y", (d: LabelEnabledDataPoint) => d.labelY
```

Comment:
Since it's less than 120 symbol we should write this function call into a single line